### PR TITLE
Restyle ui

### PR DESCRIPTION
**Pull request by Exclaminator**
Won't close any issues but the styling becomes a lot more intuitive.

### CODE REVIEW

**Comment by mpsijm** [comment]
Thank you so much for this addition :D I have one request though, can you add `cursor: default;` when the checkboxes are disabled? :) At the moment this looks a little weird XD

**Comment by Exclaminator** [comment]
Jup Ill fix it

File: resources/options.js [hunk 1]
```diff
@@ -126,14 +126,20 @@ function addOptionClickEvents() {
         optionsElement(i).click(saveOptions);
     }
 
+    updateSubOptionVisibility();
+}
+
+
```
Comment (mpsijm) [inline resources/options.js]: Is the double newline needed here?

**Comment by mpsijm** [comment]
Thanks for resolving my feedback, and fixing a bug in the process :D You have my approval of merging :)

**Comment by thervh70** [comment]
Nice change! Will merge!

**Pull request merged**